src/functional/ffront/parsing/single_static_assign.py

```diff
@@ -76,6 +95,12 @@ def visit_Assign(self, node: ast.Assign) -> ast.Assign:
         node.targets = [self.visit(target) for target in node.targets]
         return node
 
+    def visit_AnnAssign(self, node: ast.AnnAssign) -> ast.AnnAssign:
+        if node.value:
+            node.value = self.RhsRenamer(self.state).visit(node.value)
+        node.target = self.visit(node.target)
```

Comment:
Shouldn't this line be part of the previous `if`? I mean, if there is no value, it's just a variable type declaration, right?
```suggestion
            node.target = self.visit(node.target)
```

Reply:
True, it is a bit weird, that
```
a: int
a = 5
```

would turn into

```
a: int
a$0 = 5
```

But better than losing the value like in the following case:

```
a = 5
a: int
```

would have been

```
a$0 = 5
a$1: int
```

Reply:
actually I found a way to not lose the value and only lose the type annotation in the case of a dangling empty AnnAssign.